src/providers/ibmcloud.json

```diff
@@ -0,0 +1,42 @@
+{
+    "ibmcloud": {
+        "prefix": "tf-ibmcloud",
```

Comment:
tried on local machine, vscode wasn't able to provide intellisense for this - maybe it's my machine or the version of vscode. I'd suggest to change the  name and prefix to the following:
```
"ibm_provider": {
  "prefix": "tf-ibm_provider"...
}
```
that sample above seems to work on my machine.

my current setup:

vscode v1.19.2
on ubuntu Xenial

Reply:
Oke, I update it tomorrow.

Reply:
thanks @HintikkaKimmo I'll merge it right away and hopefully push it to marketplace when you get a chance to update your PR. cheers